openshift/release/artifacts/eventing-kafka-controller.yaml

```diff
@@ -1135,6 +1135,14 @@ metadata:
   labels:
     kafka.eventing.knative.dev/release: v1.7
 rules:
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
I guess this change is because we have merged a PR that didn't run `make generate-release`